content/en/docs/setup/platform-setup/gardener/index.md

```diff
@@ -70,8 +70,8 @@ Gardener dashboard or via cli as follows:
 $ kubectl --namespace shoot--my-project--my-cluster get secret kubecfg --output jsonpath={.data.kubeconfig} | base64 --decode > my-cluster.yaml
 {{< /text >}}
 
-This kubeconfig file has full administrators access to you cluster. For the rest
-of this guide be sure you have `export KUBECONFIG=my-cluster.yaml` set.
+This kubeconfig file has full administrator access to you cluster.
+For any activities with the payload cluster be sure you have `export KUBECONFIG=my-cluster.yaml` set.
```

Comment:
Not sure if you need the set. Maybe this (and similar in the prior comment) should be ... exported `KUBECONFIG=my-cluster.yaml`?

Reply:
I will leave as is for c&p.